exercises/shared/.docs/tests.md

```diff
@@ -1,14 +1,109 @@
 # Running tests
 
-Execute the tests with:
+Tests are written in test files and placed in the `test` directory and implemented
+as:
+
+```elixir
+test "short test description" do
+  ... test implementation ...
+end
+```
```

Comment:
Let's make this a snippet without syntax errors so that the syntax coloring works:
```suggestion
```elixir
test "short test description" do
  # ... test implementation ...
end
```
```